### DISCUSSION TITLE
Ender-3  V2 Just keeps Extruding while idle

### DISCUSSION THREAD

**Discussion by rsvkgit2018** [Jyers/Marlin#2000] [General Discussion]
After upgrading to Jyers 2.0.1 from the stock firmware, I noticed the printer keeps extruding even when the extruder motor is idle. This is causing problems especially when I pause and resume the print. May be it is just a coincidence with the firmware upgrade. Anybody noticed this problem?

Thanks in advance.

-Ender 3 V2, dual z motors, CR touch with Jyers 2.0.1.

**Comment by Hologos** [comment]
It's caused by the high hotend temperature. The filament is oozing, which is normal. You have to retract at the end or when you pause.
